tsdb/chunkenc/chunk_test.go

```diff
@@ -137,32 +136,27 @@ func benchmarkIterator(b *testing.B, newChunk func() Chunk) {
 				break
 			}
 			a.Append(p.t, p.v)
-			i++
 			j++
 		}
-		chunks = append(chunks, c)
 	}
 
 	b.ReportAllocs()
 	b.ResetTimer()
 
-	b.Log("num", b.N, "created chunks", len(chunks))
-
-	res := make([]float64, 0, 1024)
-
+	var res float64
 	var it Iterator
-	for i := 0; i < len(chunks); i++ {
-		c := chunks[i]
-		it := c.Iterator(it)
+	for i := 0; i < b.N; {
+		it := chunk.Iterator(it)
 
 		for it.Next() {
 			_, v := it.At()
-			res = append(res, v)
+			res = v
```

Comment:
What does this `res` achieve compared to just `_, _ = it.At()`

Reply:
Not much.  I wanted to make it similar to the previous code, which collected results in `res`.